.eslintrc.json

```diff
@@ -4,6 +4,10 @@
     "es2021": true
   },
   "extends": "eslint:recommended",
+  "parserOptions": {
+    "ecmaVersion": "latest",
+    "sourceType": "module"
+},
   "overrides": [
     {
       "files": ["**/*.cy.js", "cypress.config.js"],
```

Comment:
Missing override sections for Jest, Cypress overrides are not detailed enough. Please check this section of the content again. It contains the correct config: https://content.noroff.dev/workflow/end-to-end-testing.html

Reply:
I dont realy understood what you mean? I looked through that page, and changed mine to look like it?